app/views/admin/users/show.html.haml

```diff
@@ -7,12 +7,16 @@
     %p
       Account active since:
       = @user.created_at.to_s(:short)
+  .govuk-grid-column-one-third
+    %h2.govuk-heading-s
+      Actions
+    %p
```

Comment:
I think we need to have a think about this one, let's go with this for now and I've created a card to look at this as a component.